apps/site/src/content/blog/astro-theme-provider.mdx

```diff
@@ -0,0 +1,84 @@
+- **Inconsistent**: There is no standard for authoring templates, most templates use different combinations of patterns and structures. For example, many templates have a config file for customization, but the location of the file and its options vary depending on the template. Learning how to use a template while also learning how to use Astro can be confusing.
+
+## A better way to create themes
+
+The first step to making themes more user-friendly is shifting away from project templates and towards Astro integrations. Using an integration has many different advantages over a traditional template, for example:
+
+* Publish and use themes as a NPM package
+* Update a theme by updating its package
+* Install themes using a single command \`astro add theme\`
+* Add themes to existing projects
+* Configure a theme by passing arguments
+
+Although theme integrations are more user-friendly than project templates, it comes at the cost of being harder to create. They require a lot of boilerplate and a lower level knowledge of Astro and Vite making it inaccessible for a lot of developers.
+
+Astro Theme Provider solves this by making it easier for theme authors to create theme integrations without having to write any integration code. Themes can be authored similar to a normal Astro project and be exported for others to use with a single function call.
+
+```
+package/
+├── public
+├── src/
+│   ├── assets
+│   ├── components
+│   ├── css
+│   ├── layouts
+│   └── pages
+├── index.ts
+└── package.json
+```
+
+```ts
+// /package/index.ts
+import { z } from 'astro/zod'; 
+import defineTheme from 'astro-theme-provider';
+
+export default defineTheme({
+    schema: z.object({
+        title: z.string()
+    })
+})
+```
+
+After creating your theme, it can be used inside an Astro project as an integration:
+
+```ts
+import { defineConfig } from "astro/config";
+import myTheme from "my-theme";
+
+export default defineConfig({
+    integrations: [
+        myTheme({
+            config: {
+                title: "Welcome to my theme!",
+            },
+        }),
+    ],
+});
+```
+
+## Using the right tool for the job
+
+Theme integrations may be easier to use, but there are some cases where using a traditional project template makes more sense. For example, themes that require modifications to the source code, like a SAAS starter, are better suited as project templates. This can also make theme integrations harder to develop. Since users can't edit the source code themselves, authors have to create more verbose pages and components to support customizations from the user.
+
+Ultimately, it is up to the theme author to weigh the advantages and disadvantages of both approaches and choose the one that works best for their project.
```

Comment:
```suggestion
```

I personally would get rid of this section completely, it is important to highlight that, but it should go in the README or the docs in the future,  I don't think it should go in the announcement blog post.